apps/right-to-rent-check/translations/src/en/fields.json

```diff
@@ -1,5 +1,12 @@
 {
-  "name": {
-    "label": "Name"
+  "living-status": {
+    "options": {
+    	"yes": {
+    		"label": "Yes"
+    	},
+    	"no": {
+    		"label": "No"
+    	}
+    }
```

Comment:
It's my sublime settings, I've set them but they seem to change project to project.  I've now amended